flagsmith/analytics.py

```diff
@@ -17,7 +19,14 @@ class AnalyticsProcessor:
     the Flagsmith SDK. Docs: https://docs.flagsmith.com/advanced-use/flag-analytics.
     """
 
-    def __init__(self, environment_key: str, base_api_url: str, timeout: int = 3):
+    def __init__(
+        self,
+        environment_key: str,
+        base_api_url: str,
+        timeout: int = 3,
+        session: FuturesSession = None,
```

Comment:
Maybe we should update the type hint here? Although `FuturesSession` does quack like a `request.session` :thinking: